RELEASE_NOTES.md

```diff
@@ -126,7 +126,7 @@ See the [snap_package](https://docs.chef.io/resource_snap_package.html) document
 
 ### windows_task
 
-windows_task now supports the Start When Available option with a new `start_when_available` property.
+The `windows_task` resource now supports the Start When Available option with a new `start_when_available` property.
```

Comment:
Also, is the "Start When Available" option supposed to be all one word per https://docs.microsoft.com/en-us/windows/desktop/TaskSchd/taskschedulerschema-startwhenavailable-settingstype-element ?

Reply:
We change the casing. Microsoft loves camelcase, but that's really just a windows things.

Reply:
Understood!